app/views/houses/parties/_party.html.haml

```diff
@@ -10,6 +10,8 @@
         %h2= link_to(party.name, house_parties_party_members_current_path(@house.graph_id, party.graph_id))
         - if party.name === "Speaker"
           %p= "#{t('house_of_commons_speaker')}"
+        - elsif party.name === "Lord Speaker"
```

Comment:
This whole statement should probably be moved into the controller instead of polluting the views - I'm happy to accept if this is a business-driven fix, can we add it to the backlog?